src/modules/blockchain/svm/listeners/solana.listener.ts

```
@@ -55,22 +59,51 @@ export class SolanaListener extends BaseChainListener {
+          this.handleProverProgramLogs.bind(this),
+          'confirmed',
+        );
+
+        this.logger.log(
+          `Solana listener started for Hyper Prover program ${this.hyperProverProgramId.toString()}. Listening for IntentProven events.`,
+        );
+      } catch (error) {
+        this.logger.error(`Failed to subscribe to Hyper Prover program logs:`, toError(error));
+      }
+    } else {
+      this.logger.warn('Hyper Prover address not configured, IntentProven events will not be captured');
```

ESLint

[error] Replace `'Hyper·Prover·address·not·configured,·IntentProven·events·will·not·be·captured'` with `⏎········'Hyper·Prover·address·not·configured,·IntentProven·events·will·not·be·captured',⏎······`

(prettier/prettier)